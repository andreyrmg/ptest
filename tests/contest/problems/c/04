100000000
abc def ghi jkl mno prs tuv xyz.#