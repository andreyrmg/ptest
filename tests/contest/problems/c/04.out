99#99901002#2212223#3313334#4414445#55155566#66617#77#777188#888911